src/backbone.intercept.js

```diff
@@ -60,6 +60,12 @@ Backbone.Intercept = {
       return;
     }
 
+    // If the Ctrl key is held, we want to open in a new tab.
+    bypass = e.ctrlKey;
+    if (bypass) {
```

Comment:
I am reusing the existing `bypass` variable, and I believe it clarifies what the check actually does.